site/src/components/BuildsTable/BuildRow.tsx

```diff
@@ -0,0 +1,139 @@
+import { makeStyles } from "@material-ui/core/styles"
+import TableCell from "@material-ui/core/TableCell"
+import TableRow from "@material-ui/core/TableRow"
+import { WorkspaceBuild } from "api/typesGenerated"
+import { Stack } from "components/Stack/Stack"
+import { useTranslation } from "react-i18next"
+import { MONOSPACE_FONT_FAMILY } from "theme/constants"
+import {
+  displayWorkspaceBuildDuration,
+  getDisplayWorkspaceBuildInitiatedBy,
+} from "util/workspace"
+import { BuildAvatar } from "./BuildAvatar"
+
+export interface BuildRowProps {
+  build: WorkspaceBuild
+}
+
+export const BuildRow: React.FC<BuildRowProps> = ({ build }) => {
+  const styles = useStyles()
+  const { t } = useTranslation("workspacePage")
+  const initiatedBy = getDisplayWorkspaceBuildInitiatedBy(build)
+
+  return (
+    <TableRow
+      hover
+      data-testid={`build-${build.id}`}
+      className={styles.buildRow}
+    >
+      <TableCell className={styles.buildCell}>
+        <Stack
+          direction="row"
+          alignItems="center"
+          className={styles.buildRow}
+          tabIndex={0}
+        >
+          <Stack
+            direction="row"
+            alignItems="center"
+            justifyContent="space-between"
+            className={styles.buildRowMain}
+          >
+            <Stack direction="row" alignItems="center">
+              <BuildAvatar build={build} />
+              <div>
+                <Stack
+                  className={styles.buildResume}
+                  direction="row"
+                  alignItems="center"
+                  spacing={1}
+                >
+                  <span>
+                    <strong>{initiatedBy}</strong>{" "}
+                    {build.reason !== "initiator" ? "automatically " : " "}
+                    <strong>
+                      {t(`buildTransitionMessage.${build.transition}`)}
+                    </strong>{" "}
+                    the workspace
+                  </span>
+
+                  <span className={styles.buildTime}>
+                    {new Date(build.created_at).toLocaleTimeString()}
+                  </span>
+                </Stack>
+
+                <Stack direction="row" spacing={1}>
+                  <span className={styles.buildInfo}>
+                    Reason: <strong>{build.reason}</strong>
```

Comment:
Can we make translations for line 67 and 71?

Reply:
Sure